Bottom: 5fdbbe8b69b65ba52624c1975f602c1ea6576fd9
Top:    602b0fb2bde1455fdf016f96d88739765827b1b9
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-03 22:12:16 +0000

Refresh of play-with-locked-object

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index cd3432d..d11d5da 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -110,6 +110,8 @@ hid_gc_struct;
 static void draw_lead_user (render_priv *priv);
 static bool ghid_unproject_to_z_plane (int ex, int ey, Coord pcb_z, Coord *pcb_x, Coord *pcb_y);
 
+void ghid_set_lock_effects (hidGC gc, AnyObjectType *object);
+
 
 #define BOARD_THICKNESS         MM_TO_COORD(1.60)
 #define MASK_COPPER_SPACING     MM_TO_COORD(0.05)
